ubuntu_llvm9/Dockerfile

```diff
@@ -1,150 +0,0 @@
-# Check http://releases.llvm.org/download.html#9.0.0 for the latest available binaries
```

Comment:
Same thing for this file, please keep it!!

Reply:
I would suggest to re-add it based on 20.04? Then will also need to do less manual installation.

Reply:
It will need 100% manual installation, because nothing is built on `libc++` on ubuntu.

Reply:
As discussed, this will be handled with a new image in the future. I left the existing Dockerfile in for reference and renamed it to `ubuntu1804-llvm9` to be consistent with the remaining images.